book/content/remote_events/01/activity-types.md

```diff
@@ -0,0 +1,238 @@
+
+#### Available Tools
+
+- [Zoom](https://zoom.us) is a very popular video conferencing platform that includes screen sharing for multiple participants.
+Other benefits include support for a large number of participants, breakout rooms to facilitate discussions between smaller groups, and a chat feature.
+- [sli.do](https://www.sli.do/) is a platform for to collect questions for Q&A sessions.
+Users login to their event that can be split into sessions relating to each talk.
+They can then submit questions which can be upvoted by other users.
+A session chair can then moderate these question and pose them to the speaker.
+
+## Prototyping or Ideas Generation Session
+
+By a prototyping or an ideas generation session, which in UK academia can also be termed a sandbox, we mean a group activity focused on brain storming around a theme which can a particular problem or set of problems or on how to get the most of a new technology, technique or material etc.
+These activities tend to also act as a way to introduce people who have shared concerns but will not previously have met as they are from different disciplines or communities.
+
+### Requirements
+
+Usually, at a prototyping or an ideas generation events you have a lead or chair who co-ordinates the activity.
+The session will start with each person introducing themselves and providing details of where their interests are relevant to the theme of the session. At small events where the participants may also be asked to bring an item or items that represents them or the topic that interests them.
+At a larger event there may be too many participants to allow everyone to interduce themselves in which case there will be an introduction and selected participants will give a presentation.
+
+Once everyone understands the theme and has been introduced there will a networking session or breakout sessions.
+Generally, at this point the chair will identify topics or sub-themes and note them down so that all can see.
+
+A networking session allows people to mingle and find other participants with shared interests.
+If there are good common themes or sub-themes then the chair may help participants find participants with shared interests.
+This could be down by putting a flip chart of post-its in different parts of a room so that participants can cluster and talk.
+
+Alternatively break out sessions can be more structured and allow the participants to talk, do an activity such as making notes for a blog or notes for post-its or drawing pictures of products or art works.
+More creative sessions may provide a maker space or materials to build prototypes.
+The break-out groups can be self-organised from the ‘grass roots’ or the chair may allocate participants to groups and may event move them between a number of break out groups and sessions so that for example every participant is shares at least 1 break out session with every other participant.
```

Comment:
spl - even move them...
so that, for example, every participant shares at least...